extensions/target-specific/chromium/blink/blink-helpers/blink-helpers.js

```diff
@@ -631,6 +635,25 @@ Loader.OnLoad(function() {
             .then((sortedWebFrames) => Promise.map(sortedWebFrames, (webFrame) => webFrame.vcast().f("frame_.raw_").f("dom_window_.raw_").F("document")))
             .then((sortedDocuments) => Promise.filter(sortedDocuments, (document) => !document.isNull()))
         },
+        GetRootLayoutObjects: (typename_for_error) => {
+            return BlinkHelpers.GetDocuments()
+            .then((documents) => {
+                if (documents.length == 0) {

```

Comment:
We should probably transfer this part to GetDocuments, because the DOMTree extension also does the same thing. Anyways, this was something I should have done when I created GetDocuments, so feel free to leave it alone if you don't want to fix it. I can fix it later.

Reply:
I'll leave this one to you